deploy/helm/sumologic/values.yaml

```diff
@@ -4503,3 +4503,49 @@ opentelemetry-operator:
 
   manager:
     env:
+
+pvc-cleaner:
+  metrics:
+    enabled: true
+  logs:
+    enabled: true
```

Comment:
It should be disabled by default (per this internal discussion: https://docs.google.com/document/d/1qC471RG2YTKw3PjDaqtQ6Rt9U8UaR3yeFwkqV0ZuplA/edit#heading=h.e6ctt1cf9j1s)